.github/workflows/template_gitops.yml

```diff
@@ -58,9 +60,12 @@ jobs:
           dockerusername: ${{ secrets.docker_username }}
           dockerpassword: ${{ secrets.docker_password }}
           dockerenabled: ${{ inputs.dockerenabled }}
-          dockerbuildargs: ${{ inputs.dockerbuildargs }}
+          dockerbuildargs: |
+            ${{ inputs.dockerbuildargs }}
+            # remove token if feature is available: https://github.com/github-community/community/discussions/17554
```

Comment:
Are you sure that this will work and the comment will not be passed as a build-arg to docker?